docs/_includes/apiRef/adminCountDescription.md

```diff
@@ -0,0 +1 @@
+The number of admins assigned to a configuration.
```

Comment:
This will need to be fixed:  Can't support multi-language.  Need to use include-relative and place content under the en folder.